src/SelectUsers.vue

```diff
@@ -367,7 +367,7 @@ export default {
 }
 
 .user-not-member {
-	/* background-color: #ffebee */
+	background-color: #ffebee
```

Comment:
Your fix works !

But, with the dark mode, it's not very good...

We will find another solution :thinking: